docs/pages/develop/development-builds/introduction.mdx

```diff
@@ -1,88 +1,78 @@
+The **JavaScript bundle (`npx expo start`)** is where your app's UI code and business logic is. In a production apps there is one `main.js` bundle that is shipped with the app itself. In development this JS bundle is live reloaded from your local machine. The main role of React Native is to provide a way for the JavaScript code to access the native APIs (Image, Camera, Notifications etc). But only APIs and libraries that were bundled in the **native app** can be used.
 
-<BoxLink
-  title="Get started with development builds using EAS Build"
-  description="The easiest way to get started with development builds is to use EAS Build to build your app in the cloud, because you do not need to install any native build tools locally."
-  href="/develop/development-builds/create-a-build/"
-  Icon={BookOpen02Icon}
-/>
+Since the native app bundle is immutable, this means that when you're using Expo Go for development, you can only rely on the native code and tools that exist in Expo Go. The only way to get around it is to build your native app yourself instead of using our pre-packaged sandbox. This is exactly what a **Development Build is: your own version of Expo Go** where you are free to use any native libraries and change any native config.
 
-<BoxLink
-  title="Get started with development builds in a local development environment"
-  description="This guide covers the basics of building your app for development, and provides instructions on how to configure your environment."
-  href="/guides/local-app-development/"
-  Icon={BookOpen02Icon}
+<VideoBoxLink
+  videoId="7J8LRpja9_o"
+  title="Getting the most out of Expo Development Builds"
+  description="In this talk Kadi explains the history of how Expo Go came to be, it's benefits and limitations."
 />
 
-<BoxLink
-  title="Develop an app with Expo"
-  description="An overview of the development process of building an Expo app to help build a mental model of the core development loop. See where development builds fit in."
-  href="/workflow/overview/"
-  Icon={BookOpen02Icon}
-/>
+## What can't you do in Expo Go and why
 
-<BoxLink
-  title="Getting the most out of Expo Development Builds"
-  description="A YouTube video that explains what a development build is, why it is important, and how to use it."
-  href="https://www.youtube.com/watch?v=7J8LRpja9_o"
-  Icon={VideoRecorderIcon}
+Migrating from Expo Go to a Development Build can be a bit overwhelming the first time. Expo Go is the perfect tool for learning, prototyping and experimenting, but most production apps will convert to using Development builds sooner rather than later. It helps to know exactly what is _not_ possible in Expo Go and _why_, so you can make an informed decision on when and why to make this move.
```

Comment:
```suggestion
Migrating from Expo Go to a development build can be a bit overwhelming the first time. Expo Go is the perfect tool for learning, prototyping, and experimenting, but most production apps will convert to using Development builds sooner rather than later. It helps to know exactly what is _impossible_ in Expo Go and _why_, so you can make an informed decision on when and why to make this move.
```

Just a thought: I think the first line should be more of a positive message. This sentence implies that migration is going to be hard, which might not be a good first impression considering the type of the audience you described in our last conversation. Right now, it seems that the developer is being forced to use a development build instead of Expo Go.

Reply:
Sure, I'll just remove the first sentence.